packages/whemotion-admin/.versionrc

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
this `.versionrc` files helps the release to set titles to each section of the generated changelog